app/javascript/vue/race-title.js

```diff
@@ -94,7 +88,7 @@ export default {
           method: 'PATCH',
           headers: headers,
           body: JSON.stringify({
-            category_id: this.categoryId,
+            category_id: (this.category && this.category.id) || null,
```

Comment:
Do we know if we can do the safe navigation operator yet?

Reply:
Not yet -- we could [manually enable][1] "experimental support" for it, but I'd rather wait for full support which is supposed to be this year anyway: https://github.com/tc39/proposals/blob/master/finished-proposals.md

[1]: https://github.com/babel/babel/tree/master/packages/babel-plugin-proposal-optional-chaining

Reply:
I agree, waiting for full support sounds like the best option.